### DISCUSSION TITLE
Assets folder not loading after deployment

### DISCUSSION THREAD

**Discussion by jpablou** [alshedivat/al-folio#1539] [Q&A]
Hi everyone, 

I followed the guideline and I was able to run the theme successfully in local. 

However, after deploying to github pages using the gh-pages branch in set up in Settings, the site loads everything except the assets folder. All JS and CSS files are throwing 404 errors. 

I checked my repo and I have everything in place - except main.css (has not been generated for some reason). 

Any ideas?

Site: https://jpablou.github.io/personal-site/

**Comment by george-gca** [comment]
Is it happening from the start? Have you tried it the version straight from the repo, without further modifications?
